.github/CODEOWNERS

```
@@ -13,6 +13,7 @@ posthog/tasks/stop_surveys_reached_target.py @PostHog/team-surveys
 posthog/tasks/update_survey_adaptive_sampling @PostHog/team-surveys
 posthog/tasks/update_survey_iteration @PostHog/team-surveys
 posthog/tasks/test/test_stop_surveys_reached_target.py @PostHog/team-surveys
+posthog/templates/surveys/public_survey.html @PostHog/team-surveys
```

```suggestion
posthog/templates/surveys @PostHog/team-surveys
```

so errors.html is part of it